terraform/environments/sprinkler/iam.tf

```diff
@@ -0,0 +1,68 @@
+resource "aws_iam_role" "github_actions" {
```

Comment:
This role has already been created now in the account you don't need to created it here again

Reply:
done